website/payments/templates/admin/payments/change_list.html

```diff
@@ -0,0 +1,11 @@
+{% extends "admin/change_list.html" %}
+{% load i18n admin_urls static compress %}
+
+{% block object-tools-items %}
+    {% if opts.model_name == "payment" or opts.model_name == "batch" %}
+    <li>
+        <a href="{% url 'admin:payments_batch_new_batch_filled' %}">{% trans "New batch from unbatched Thalia Pay payments" %}</a>
```

Comment:
Maybe only display this button if there are unbatched Thalia Pay payments

Reply:
I'm closing this because it is not really necessary and a lot of work